### PR TITLE
Improve styling for visualization page

### PR DESCRIPTION
**Pull request by edlu77**

### CODE REVIEW

File: apps/cde-ui/src/app/pages/visualization/visualization.component.scss [hunk 1]
```diff
@@ -1,3 +1,6 @@
 :host {
   display: block;
+  min-width: 1280px;
```
Comment (axdanbol) [inline apps/cde-ui/src/app/pages/visualization/visualization.component.scss:3]: Why?

File: libs/cde-visualization/src/lib/cde-visualization/cde-visualization.component.html [hunk 2]
```diff
@@ -2,6 +2,7 @@
   <cde-visualization-header
     (downloadNodes)="downloadNodes()"
     (downloadEdges)="downloadEdges()"
+    homeLink="home"
```
Comment (axdanbol) [inline libs/cde-visualization/src/lib/cde-visualization/cde-visualization.component.html:5]: The cde-visualization is supposed to take this as an input, not hardcoded in it's html...

File: libs/cde-visualization/src/lib/components/histogram/histogram.component.scss [hunk 3]
```diff
@@ -35,3 +36,9 @@
     flex-grow: 1;
   }
 }
+
+::ng-deep {
+  .histogram-panel {
+    max-width: 27.5rem;
```
Comment (axdanbol) [inline libs/cde-visualization/src/lib/components/histogram/histogram.component.scss:42]: Why?

File: libs/cde-visualization/src/lib/components/histogram/histogram.component.ts [hunk 4]
```diff
@@ -1,14 +1,15 @@
+import { OverlayModule } from '@angular/cdk/overlay';
 import { CommonModule, DOCUMENT } from '@angular/common';
 import {
   ChangeDetectionStrategy,
   Component,
-  ElementRef,
-  Renderer2,
   computed,
   effect,
+  ElementRef,
```
Comment (axdanbol) [inline libs/cde-visualization/src/lib/components/histogram/histogram.component.ts:8]: Remove unused imports

File: libs/cde-visualization/src/lib/components/visualization-header/visualization-header.component.html [hunk 5]
```diff
@@ -69,10 +73,16 @@
       </ng-template>
     </ng-container>
   </span>
-  <button mat-flat-button color="primary" class="mat-caption">
+  <a
+    mat-flat-button
+    color="primary"
+    class="mat-caption"
+    href="https://github.com/hubmapconsortium/hra-ui/blob/cell-distance-explorer/libs/cde-visualization/README.md"
```
Comment (axdanbol) [inline libs/cde-visualization/src/lib/components/visualization-header/visualization-header.component.html:80]: Should point to an EMBEDDING.md

**Review by axdanbol — CHANGES_REQUESTED** [review]

File: apps/cde-ui/src/app/pages/visualization/visualization.component.scss (hunk 1 above)
Comment (edlu77) [reply apps/cde-ui/src/app/pages/visualization/visualization.component.scss:3]: According to the spec any size smaller than 1280px will use a scroll bar instead

File: libs/cde-visualization/src/lib/components/histogram/histogram.component.scss (hunk 3 above)
Comment (edlu77) [reply libs/cde-visualization/src/lib/components/histogram/histogram.component.scss:43]: The other tooltips have this to limit the width of the dialog, should I take it out?

File: libs/cde-visualization/src/lib/components/node-dist-visualization/node-dist-visualization.component.scss [hunk 6]
```diff
@@ -1,11 +1,12 @@
 :host {
   display: block;
   position: relative;
-  height: calc(100% - 8px);
   padding: 0.25rem;
   background-color: black;
   isolation: isolate;
   border-radius: 0.5rem;
+  height: 0;
```
Comment (axdanbol) [inline libs/cde-visualization/src/lib/components/node-dist-visualization/node-dist-visualization.component.scss:8]: Won't this affect the visualization itself? It seems a bit strange to set the height to 0

File: libs/cde-visualization/src/webcomponent-example.html [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+      const nodeHover = (nodeEntry) => {
+        console.log('node hover', nodeEntry);
+      };
+      ftu.addEventListener('nodeClicked', nodeClick);
+      ftu.addEventListener('nodeHovered', nodeHover);
+    });
+  </script>
+  <body>
+    <div class="wc-content">
+      <cde-visualization-root
```
Comment (axdanbol) [inline libs/cde-visualization/src/webcomponent-example.html:31]: Please provide actual values for these attributes. Nodes, edges, color-map, and metadata all accept urls to files

**Review by axdanbol — CHANGES_REQUESTED** [review]
A few more tweaks

File: apps/cde-ui/example-data/nodes.csv
[the remark this answers is not in the document]
Comment (bherr2) [reply apps/cde-ui/example-data/nodes.csv:1]: Let's use a smaller dataset from the hra-node-dist-vis datasets. `skin-celldive-ge / region_1 (2,868 cells)` looks pretty good. <https://cdn.humanatlas.io/image-store/vccf-data-cell-nodes/published/skin-celldive-ge/region_1-nodes.csv>

**Review by axdanbol — APPROVED** [review]

**Pull request merged**